js/zb.js

```diff
@@ -969,20 +969,49 @@ module.exports = class zb extends Exchange {
     }
 
     async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
-        if (type !== 'limit') {
-            throw new InvalidOrder (this.id + ' allows limit orders only');
-        }
         await this.loadMarkets ();
-        const request = {
-            'price': this.priceToPrecision (symbol, price),
-            'amount': this.amountToPrecision (symbol, amount),
-            'tradeType': (side === 'buy') ? '1' : '0',
-            'currency': this.marketId (symbol),
-        };
-        const response = await this.spotV1PrivateGetOrder (this.extend (request, params));
+        const market = this.market (symbol);
+        if (market['spot']) {
+            if (type !== 'limit') {
```

Comment:
* `if (type === `market`)` 

1 and 0 should still be valid types

Reply:
This is leftover logic from handling the spot API and the numbers aren't valid types for the spot API

Reply:
From below it looks like `tradeType` can be `1` or `0`

Reply:
I think tradeType deals with the side for spot markets, I'm not sure what you're suggesting

Reply:
Well the exchange calls it `tradeType`, so a user might confuse our type, with type on the exchange, so I think we should accept the types `1` and `0`

Reply:
Okay tradeType deals with the side argument though not type and I'm a bit skeptical of replacing igors spot implementation here but I think I can delete this error check and just leave the if (type === market) check below for both spot and swap

Reply:
maybe just check that it's either `limit` `1` or `0`

Reply:
@kroitor